### PR TITLE
Adding TraceablePresentation to OAS

### PR DESCRIPTION
**Pull request by nissimsan**
Ref: https://github.com/w3c-ccg/traceability-interop/issues/325

### CODE REVIEW

File: docs/openapi/schemas/VerifiablePresentation.yml [hunk 1]
```diff
@@ -10,9 +10,23 @@ allOf:
...
-    "@context": ["https://www.w3.org/2018/credentials/v1"],
+    '@context': [
+      'https://www.w3.org/2018/credentials/v1',
+      'https://w3id.org/traceability/v1'
+    ],
     "id": "urn:uuid:07aa969e-b40d-4c1b-ab46-ded252003ded",
-    "type": ["VerifiablePresentation"],
+    'type': [
+      'VerifiablePresentation',
+      'TraceablePresentation'
```
Comment (OR13) [inline docs/openapi/schemas/VerifiablePresentation.yml:20]: This schema should not be a "Traceable Presentation" it should match normative requirements of the W3C TR.

**Review by OR13 — CHANGES_REQUESTED** [review]
Do not change the example of a normal verifiable presentation.

File: docs/openapi/schemas/Workflow.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    'type': ['Workflow'],
+    "workflow":{
+      "instance":[
+        "urn:uuid:f5fb6ce4-b0b1-41b8-89b0-331ni58b7ee0"
+      ],
+      "definition":[
+        "urn:uuid:n1552885-cc91-4bb3-91f1-5466a0be084e"
+      ]
+    }
+  }
```
Comment (brownoxford) [inline docs/openapi/schemas/Workflow.yml:24]:
```suggestion
  {
    "@context": ["https://w3id.org/traceability/v1"],
    "type": ["Workflow"],
    "workflow":
      {
        "instance": ["urn:uuid:f5fb6ce4-b0b1-41b8-89b0-331ni58b7ee0"],
        "definition": ["urn:uuid:n1552885-cc91-4bb3-91f1-5466a0be084e"],
      },
  }
```

Simple whitespace and quote consistency change request

File: docs/openapi/schemas/VerifiablePresentation.yml [hunk 3]
```diff
@@ -10,23 +10,9 @@ allOf:
...
-    ],
-    "workflow":{
-      "instance":[
-        "urn:uuid:f5fb6ce4-b0b1-41b8-89b0-331ni58b7ee0"
-      ],
-      "definition":[
-        "urn:uuid:n1552885-cc91-4bb3-91f1-5466a0be084e"
-      ]
-    },
+    'type': ['VerifiablePresentation'],
```
Comment (brownoxford) [inline docs/openapi/schemas/VerifiablePresentation.yml]:
```suggestion
    "type": ["VerifiablePresentation"],
```

File: docs/openapi/schemas/VerifiablePresentation.yml [hunk 4]
```diff
@@ -10,23 +10,9 @@ allOf:
         $ref: "./PresentationLinkedDataProof.yml"
 example:
   {
-    '@context': [
-      'https://www.w3.org/2018/credentials/v1',
-      'https://w3id.org/traceability/v1'
-    ],
+    '@context': ['https://www.w3.org/2018/credentials/v1'],
```
Comment (brownoxford) [inline docs/openapi/schemas/VerifiablePresentation.yml]:
```suggestion
    "@context": ["https://www.w3.org/2018/credentials/v1"],
```

File: docs/openapi/schemas/TraceablePresentation.yml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    required:
+    properties:
+      workflow:
+        $ref: "./Workflow.yml"
+example:
+  {
+    '@context': [
+      'https://www.w3.org/2018/credentials/v1',
+      'https://w3id.org/traceability/v1'
+    ],
```
Comment (brownoxford) [inline docs/openapi/schemas/TraceablePresentation.yml:15]:
```suggestion
    "@context": [
      "https://www.w3.org/2018/credentials/v1",
      "https://w3id.org/traceability/v1"
    ],
```

File: docs/openapi/schemas/TraceablePresentation.yml [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    "id": "urn:uuid:07aa969e-b40d-4c1b-ab46-ded252003ded",
+    'type': [
+      'VerifiablePresentation',
+      'TraceablePresentation'
+    ],
```
Comment (brownoxford) [inline docs/openapi/schemas/TraceablePresentation.yml:20]:
```suggestion
    "type': [
      "VerifiablePresentation",
      "TraceablePresentation"
    ],
```

**Review by brownoxford — CHANGES_REQUESTED** [review]

**Review by OR13 — APPROVED** [review]

**Comment by nissimsan** [comment]
I'll fix the failure for next time.

File: docs/openapi/schemas/TraceablePresentation.yml [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+      workflow:
+        $ref: "./Workflow.yml"
+example:
+  {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "urn:uuid:07aa969e-b40d-4c1b-ab46-ded252003ded",
+    "type': [
```
Comment (brownoxford) [inline docs/openapi/schemas/TraceablePresentation.yml:17]:
```suggestion
    "type": [
```

**Review by brownoxford — CHANGES_REQUESTED** [review]

**Comment by nissimsan** [comment]
No worries, @brownoxford. The quote helped. :)

**Review by brownoxford — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Pull request merged**